mapentity/middleware.py

```diff
@@ -21,8 +21,9 @@
     socket.gethostbyname(CAPTURE_SERVER_HOST),
 ]
 for interface in interfaces():
-    for link in ifaddresses(interface)[AF_INET]:
-        AUTOLOGIN_IPS.append(link['addr'])
+    for link in ifaddresses(interface).get(AF_INET, None):
+        if link:
+            AUTOLOGIN_IPS.append(link['addr'])
```

Comment:
Ça marche ??? J'ai l'impression que ça itère sur None. J'aurai plutôt mis [] que None comme valeur par défaut. Et pas mis de if.

Reply:
Ou alors:
```
links = ifaddresses(interface).get(AF_INET)
if links:
    for link in links:
```

Mais la 1ère solution est plus simple.